include/follow.php

```diff
@@ -82,6 +82,11 @@ function new_contact($uid,$url,$interactive = false) {
 		return $result;
 	}
 
+	if (blocked_url($url)) {
+		$result['message'] = t('Blocked domain');
```

Comment:
I'm still not sure about how to handle this, I'd rather this got out of the door as fast as possible, notice could use a verbosity overhaul, but I don't think it's in the scope of this PR.